templates/monitoring-docker-compose.yml.j2

```diff
@@ -9,8 +9,26 @@ services:
       - {{ monitoring_nexporter_docker_bind }}
     volumes:
       - "/:/host:ro,rslave"
-    healthcheck:
-      test: 'curl {{ monitoring_nexporter_docker_bind }}/metrics'
-      interval: 2s
-      timeout: 20s
+#    healthcheck:
+#      test: 'curl http://{{ monitoring_nexporter_host }}:{{ monitoring_nexporter_port }}/metrics'
+#      interval: 2s
+#      timeout: 20s
+{% endif %}
+{% if monitoring_promtail_enabled %}
+  promtail:
+    image: grafana/promtail:{{ monitoring_promtail_image_version }}
+    container_name: {{ monitoring_promtail_container_name }}
+    restart: always
+    ports:
+      - {{ monitoring_promtail_docker_bind}}
+    volumes:
+      - {{ monitoring_promtail_config_dir }}:{{ monitoring_promtail_config_dir }}
+{% for volume in monitoring_promtail_docker_volumes %}
+      - {{ volume }}
+{% endfor %}
+#    healthcheck:
+#      test: 'curl http://{{ monitoring_promtail_host }}:{{ monitoring_promtail_port }}/ready'
```

Comment:
Can you think about a reason why this healthcheck was failing?
Node exporter returns a 200 if done manually, yet docker says unhealthy.

On the other hand, promtail is listening to the port, but returns empty replies, with no HTTP content (no headers, error code, etc), and then keeps restarting.

cc/ @sauloperez @danypr92

Reply:
The `ready` endpoint is documented [here](https://github.com/grafana/loki/tree/master/docs/clients/promtail#api)

Reply:
then something is really broken if it does not return HTTP things. Should we create an issue for it?